screens/Home/MyVcs/AddVcModalMachine.ts

```diff
@@ -349,6 +368,15 @@ export const AddVcModalMachine =
           );
           return response.response.requestId;
         },
+
+        calculateHashedId: async (context) => {
+          const salt =
+            '1234567891011121314151617181920212223242526272829303132333435363';
```

Comment:
Can we move salt to constant or argon2iConfigForUinVid?

Reply:
committed this change